solarforecastarbiter/reports/templates/html/body.html

```diff
@@ -45,15 +55,31 @@ <h2 id="report-metadata">Report Metadata</h2>
   <li>End: {{ report.report_parameters.end }}</li>
   <li>Generated at: {{ report.raw_report.generated_at }}</li>
 </ul>
-{% endblock %}
 
-{% block errors %}
-{% if report.raw_report.messages| length > 0 %}
-<div class="alert alert-warning">
-  {% for mesg in report.raw_report.messages %}
-  <h5>{{ mesg.message }}</h5>
-  {% endfor %}
+{% set costs = report.report_parameters.costs %}
+{% if costs | length > 0 %}
+<p>
+  Below are the cost parameters defined for this report.  Each
+  forecast-observation pair may use one set of cost parameters to
+  calculate the cost metric.
```

Comment:
I worry that a reader will think that one of the handful of costs is applied to each forecast, rather than all of them being applied to all forecasts (at least for now).

Reply:
um, one cost is applied to each forecast. Even if there are multiple, you only select one for the pair to be used in the metrics. I can reword